packages/components/src/components/counter/counter.tsx

```diff
@@ -1,14 +1,11 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 
-const Counter = ({ className, count }) => {
-    return <div className={classNames('dc-counter', className)}>{count}</div>;
+type TCounterProps = HTMLDivElement & {
```

Comment:
@iman-fs, the definition should be `HTMLProps<HTMLDivElement> & {
   count: number;
}`